packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx

```diff
@@ -62,7 +61,7 @@ it('renders the banner appropriately for a person with gestational and post mens
       gender="Male"
       gestationalAge="April 5, 2016"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```

Comment:
Should this include the img src?
e.g. `<img alt="My Cat" src={demographicsCat} />`